internal/services/disbursement_management_service.go

```diff
@@ -144,6 +149,37 @@ func (s *DisbursementManagementService) StartDisbursement(ctx context.Context, d
 			return fmt.Errorf("error updating disbursement status to started for disbursement with id %s: %w", disbursementID, err)
 		}
 
+		// 7. Produce event to send payments to TSS
+		payments, err := s.models.Payment.GetReadyByDisbursementID(ctx, dbTx, disbursementID)
+		if err != nil {
+			return fmt.Errorf("getting ready payments for disbursement with id %s: %w", disbursementID, err)
+		}
+
+		if len(payments) == 0 {
+			log.Ctx(ctx).Infof("no payments ready to pay for disbursement ID %s", disbursementID)
+			return nil
+		}
+
+		msg, err := events.NewMessage(ctx, events.PaymentReadyToPayTopic, disbursementID, events.PaymentReadyToPayDisbursementStarted, nil)
+		if err != nil {
+			return fmt.Errorf("creating new message: %w", err)
+		}
+
+		paymentsReadyToPay := schemas.EventPaymentsReadyToPayData{TenantID: msg.TenantID}
+		for _, payment := range payments {
+			paymentsReadyToPay.Payments = append(paymentsReadyToPay.Payments, schemas.PaymentReadyToPay{ID: payment.ID})
+		}
+		msg.Data = paymentsReadyToPay
+
+		if s.eventProducer != nil {
+			err := s.eventProducer.WriteMessages(ctx, *msg)
+			if err != nil {
+				return fmt.Errorf("writing message %s on event producer: %w", msg, err)
+			}
+		} else {
+			log.Ctx(ctx).Errorf("event producer is nil, could not publish message %s", msg.String())
+		}
+
```

Comment:
💡 🤔 Do you think we could create a helper to avoid repeating this in 3 different places? Personally I can't think of a place to put this helper so maybe it is not a good idea, but maybe you can figure out an easy way to do it. 

I don't think it's worth spending much time on it though, so I suggest you only invest time on it if you think it's easy peasy.